packages/mgt-chat/src/components/UnsupportedContent/UnsupportedContent.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+import { makeStyles, shorthands } from '@fluentui/react-components';
+import React from 'react';
+import { registerAppIcons } from '../styles/registerIcons';
+import { ArrowSquareUpRight24Regular } from '@fluentui/react-icons';
+registerAppIcons();
+
+const useStyles = makeStyles({
+  container: {
+    backgroundColor: '#ebebeb',
+    display: 'flex',
+    boxShadow: '0px 4px 8px 0px rgba(0, 0, 0, 0.14), 0px 0px 2px 0px rgba(0, 0, 0, 0.12)',
+    textDecorationLine: 'none',
+    color: '#424242',
+    ...shorthands.margin('4px 0 0 0'),
+    ...shorthands.borderRadius('6px'),
+    ...shorthands.padding('16px'),
+    ...shorthands.gap('6px'),
+    ':hover': {
+      backgroundColor: '#fafafa'
+    },
+    ':visited': {
+      color: '#424242'
+    }
+  },
+  cta: {
+    fontFamily: 'Segoe UI',
+    fontSize: '12px',
+    fontStyle: 'normal',
+    fontWeight: '400',
+    lineHeight: '16px',
+    textDecorationLine: 'none'
+  }
+});
+
+const UnsupportedContent = () => {
+  const styles = useStyles();
+  return (
+    // TODO: update this URL to the correct value.
+    <a className={styles.container} href="https://teams.microsoft.com">
```

Comment:
This link should be the deep link either to the chat this message is part of or (if possible to the message itself)
That means we'll need to pass at a minimum a targetUrl prop to this component.
Also, the link should use target="blank" to ensure that it opens in a new tab/window

Reply:
Likely the link target could be part of the Unsupported message type.

Reply:
For unsupported content like code snippets, loop components, you can have the raw link to the resource. However, it's a painful extraction process that involves parsing a JSON string from the attachment and getting the URL from the object. The URL property name is not the same for the different resources i.e. loop component has `componentUrl` while code snippets have `codeSnippetUrl`. I settled on building the much simpler conversation url `https://teams.microsoft.com/_#/conversations/${chatId}?ctx=chat`